src/PostgREST/Auth.hs

```diff
@@ -102,12 +104,12 @@ middleware appState app req respond = do
   conf <- getConfig appState
   time <- getTime appState
 
-  let token = fromMaybe "" $ Wai.extractBearerAuth =<< lookup HTTP.hAuthorization (Wai.requestHeaders req)
-  authResult <- runExceptT $
-    parseToken conf (LBS.fromStrict token) time >>=
-    parseClaims conf
+  let token  = fromMaybe "" $ Wai.extractBearerAuth =<< lookup HTTP.hAuthorization (Wai.requestHeaders req)
+      parseJwt = runExceptT $ parseToken conf (LBS.fromStrict token) time >>= parseClaims conf
 
-  let req' = req { Wai.vault = Wai.vault req & Vault.insert authResultKey authResult }
+  (dur,authResult) <- timeItT parseJwt
```

Comment:
This always runs right? Regardless of `configDbPlanEnabled`. I think it should be conditional to that config.